templates/main/base.html

```diff
@@ -37,5 +37,8 @@
     {% endblock %}
 {% endif %}
 {% block content %}{% endblock %}
+<footer>
+    <p>Developed collaboratively on <a href="https://github.com/discord-python/site">Github</a> • 2018</p>
```

Comment:
It's generally good practise to use HTML entities with unusual characters. Helps avoid possible encoding issues.